packages/align-deps/src/presets/microsoft/react-native/profile-0.73.ts

```diff
@@ -49,6 +49,10 @@ export const profile: Profile = {
     version: "^0.73.0",
     devOnly: true,
   },
+  clipboard: {
+    name: "@react-native-clipboard/clipboard",
+    version: "^1.10.0",
```

Comment:
Shouldn't this be 1.14? Also, can you move it to the bottom section?
```suggestion
    version: "^1.14.0",
```

Reply:
Derp, yeah. Thank you for catching all of my typos. I must have committed the wrong version as I was looking for where to put this. I thought we'd want it alphabetical but I can move to bottom.